File: app/src/main/java/bank/jaeyoung/com/bank/Calculator.java
package bank.jaeyoung.com.bank;

import java.text.DecimalFormat;

/**
 * Created by jaeyoung lee on 2018. 10. 18..
 */
public class Calculator {
	public static String calculatePreviousBalance(String balance, String amount) {
		DecimalFormat df = new DecimalFormat("#.00");
		return df.format(Double.parseDouble(balance) - Double.parseDouble(amount));
	}
}

File: app/src/main/java/bank/jaeyoung/com/bank/Calculator.kt
package bank.jaeyoung.com.bank

import java.text.DecimalFormat

/**
 * Created by jaeyoung lee on 2018. 10. 18..
 */
object Calculator {
    fun calculatePreviousBalance(balance: String, amount: String): String {
        val df = DecimalFormat("#.00")
        return df.format(java.lang.Double.parseDouble(balance) - java.lang.Double.parseDouble(amount))
    }
}
